<launch>
    <node name="keyboard2mag"
        pkg ="plane_camera_magnet"
        type = "keyboard2mag"
        output ="screen"
        launch-prefix = "gdb -ex run --args">
    </node>
    <node name="roboclaw4listen_pub"
        pkg ="plane_camera_magnet"
        type = "roboclaw4listen_pub.py"
        output = "screen">
        <remap from="/closedloopfbpoint/roboclawcmddesired" to="/keyboard2mag/roboclawcmddesired"/>
    </node>
</launch>



